docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
+but this method is less secure than creating and using an API key.
+We recommend taking an extra moment to follow the section below.
+:::
+
+### Set an API Key
+
+Generate a new API key through your account settings in the Regression Games client, and copy the generated value to your clipboard.
+
+![Generate a new API key](../core-concepts/img/api-keys/create-new-api-key.gif)
+
+:::info
+To learn more about our API keys, see the [Authenticating with API Keys](../core-concepts/authenticating-with-api-keys) section.
+:::
+
+Do not submit any information to the welcome prompt in Unity.
+Instead, close the prompt and navigate to `Edit` > `Project Settings` > `Regression Games`.
+Paste the key you just generated into the `RG API Key` field, then close the project settings.
+
+![Set an API Key in Unity](../core-concepts/img/api-keys/set-api-key-unity.png)
+
+## Restart Unity
+
+Now is a good time to restart Unity and any code editor you may have open (e.g. Rider or Visual Studio).
+Without this step, Unity and your code editor may not properly resolve your new packages even after Unity recompiles the scripts.
+
+## Add the Regression Games Overlay to Your Scene
+
+Our core package provides an in-game overlay that acts as a UI for features such as performing
+[input recording and playback](../core-concepts/gameplay-sessions/gameplay-sessions-getting-started)
+and constructing and running [Bot Sequences](../core-concepts/bot-sequences/getting-started-with-bot-sequences-and-segments),
+but it also drives our data collection behind-the-scenes.
+This means the overlay prefab **must** be present in each scene where you want to use any Regression Games features.
+However, you can hide the visibility of the overlay through the project settings under
+`Edit` > `Project Settings` > `Regression Games` if needed.
+
+Find the `RGOverlayCanvas` prefab by navigating to
+`Packages` > `Regression Games Unity Bots` > `Runtime` > `Prefabs` and dragging it into your scene's object hierarchy.
+Make sure that the canvas is at the top of the hierarchy, so that it is always visible and interactable.
+Repeat this for each scene that must have access to Regression Games features.
+
+![Screenshot of the search pane for the RGOverlayCanvas](../partials/img/add-overlay-to-scene.gif)
+
+:::info
+Your scene must have an EventSystem to make interaction with the overlay possible.
+You can add one by right-clicking in your scene's Hierarchy and selecting `GameObject` > `UI` > `Event System`.
+If you are using any of the sample scenes we provide in our guides, this is already included.
+:::
+
+
+## Next Steps
+
+The Regression Games SDK is now set up and ready to be explored.
+Visit our documentation on the [Core Concepts](../core-concepts/authenticating-with-api-keys) that drive features in our SDK and client,
+or head over to the [Guides](../guides/writing-advanced-validations) section for walkthroughs and examples of real-world applications.
```

Comment:
I think at somepoint we should be opinionated in here and tell the users which is the best next step is. (in addition to giving them these options). Given that the next page is Creating your first automated test, that might be added here explicitly.

Reply:
Once some more guides are made (waiting on me), I can go and link a few here :)

Reply:
Good point. This now directs the user to the creating-your-first-automated-test page and our guides sections explicitly